tests/cpu/compare_with_acts_seeding.cpp

```diff
@@ -247,6 +247,33 @@ TEST_P(CompareWithActsSeedingTests, Run) {
     // create grid with bin sizes according to the configured geometry
     std::unique_ptr<Acts::SpacePointGrid<SpacePoint>> grid =
         Acts::SpacePointGridCreator::createGrid<SpacePoint>(gridConf);
+
+    const auto numLocalBins = grid->numLocalBins();
+    // Currently traccc is using grid2. check if acts grid dimensionality is the
+    // same
+    EXPECT_EQ(numLocalBins.size(), 2);
```

Comment:
Since this variable is only used here, let's not make it into an lvalue. Let's just write:

```c++
EXPECT_EQ(grid->numLocalBins().size(), 2);
```

Sidenote: The C\+\+23 literal for `std::size_t` (for this case: `2UZ`) can not get here quickly enough... Since note that this particular test would fail like this with MSVC. But we're not going to build traccc with MSVC, so this is just something to note, not something to worry about...